### PR TITLE
[15.0][MIG] commission, sale_commission : Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This is the migration of sale_commission to 15.0.

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
I feel that the split is not done in an optimal way although I haven't done a thorough check. For example, shouldn't settlement be part of the commission module instead of sale_commission?

There is also this comment https://github.com/OCA/commission/issues/328#issuecomment-986704280 from @pedrobaeza of intending to have `commission`, `account_commission` and `sale_commission` out of 14.0 sale_commission. I think we should go in line with this.

File: commission/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Commissions",
+    "version": "15.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "depends": ["product", "sale_management"],
```
Comment (yostashiro) [inline commission/__manifest__.py:7]:
```suggestion
    "depends": ["product", "account"],
```

It shouldn't depend on sale_management.  Menu items should be moved under invoicing.

**Comment by ValentinVinagre** [comment]
> I feel that the split is not done in an optimal way although I haven't done a thorough check. For example, shouldn't settlement be part of the commission module instead of sale_commission?
> 
> There is also this comment [#328 (comment)](https://github.com/OCA/commission/issues/328#issuecomment-986704280) from @pedrobaeza of intending to have `commission`, `account_commission` and `sale_commission` out of 14.0 sale_commission. I think we should go in line with this.

@pedrobaeza what do u think?

**Comment by AungKoKoLin1997** [comment]
https://github.com/OCA/commission/pull/374

**Pull request closed without merging**